File: esp/mongo.js
const { MongoClient } = require('mongodb');
const uri = "mongodb+srv://penda_18:Yayefall99@cluster1.dqbti.mongodb.net/test?retryWrites=true&w=majority";
const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
client.connect(err => {
  const collection = client.db("test").collection("devices");
  // perform actions on the collection object
  client.close();
});
FROM node:alpine
COPY . /app
WORKDIR /app
CMD node app.js

File: esp/mongo.java
ConnectionString connectionString = new ConnectionString("mongodb+srv://penda_18:Yayefall99@cluster1.dqbti.mongodb.net/test?retryWrites=true&w=majority");
MongoClientSettings settings = MongoClientSettings.builder()
        .applyConnectionString(connectionString)
        .build();
MongoClient mongoClient = MongoClients.create(settings);
MongoDatabase database = mongoClient.getDatabase("test");
